CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 `Unreleased`_
 =============
 
+Changed
```

Comment:
It looks like we have a duplicate changed section here.

Reply:
Conflict fixed correctly this time and repushed